README.md

```diff
@@ -322,6 +322,14 @@ Both, trigger and output, can connect to a secure Kafka broker. The following at
 
 Username and password should reference a Azure function configuration variable and not be hardcoded.
 
+## Language support configuration
+
+For the non-C# languages, you can specify `cardinality` for choosing if the KafkaTrigger is executed in batch. 

```

Comment:
For the C# if I added the cardinality value to the host.json will it work? Or C# support only reflections to support this feature.

Reply:
@amamounelsayed  For the CSharp, Functions Host reference the type of the parameter, so that it automatically identify if it is array or not. So that we don't need to specify it on C#.

Reply:
Synced oflfline, thank you @TsuyoshiUshio 
This the same behavior is the same between Kafka and EventHub.